src/app/sandbox/mapbox/page.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import Map, { Layer, Source } from "react-map-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+function MapboxTest() {
+  return (
+    <Map
+      mapboxAccessToken="pk.eyJ1IjoiZGV2c2VlZCIsImEiOiJjbHo5NWh0d3QwMDU0MnBvanAwMmRzYWl0In0.lLN2Ur91TbYhV520kQ3SrA"
```

Comment:
@oliverroick if we promote this exercise to the real app lets move this token to the .env file.